docs/sw/fpga_tools/pci_device/pci_device.md

```diff
@@ -97,6 +104,10 @@ drivers.
 
     Unbind the driver bound to the device.
 
+    `unplug`
+
+    Prepare a host-side device for an imminent RSU event by configuring its root port and associated endpoints.
```

Comment:
How about
Remove device from PCI bus in anticipation of a RSU event by configuring its root port and associated endpoints.